src/totp/sha1hmac.ts

```diff
@@ -0,0 +1,51 @@
+import { Hash } from '@bsv/sdk';
+
+/**
+ * SHA1HMAC is implemented based on SHA256HMAC from @bsv/sdk.
+ * TODO: Consider to move it to @bsv/sdk in the future.
+ */
+export class SHA1HMAC {
+  inner: Hash.SHA1;
+  outer: Hash.SHA1;
+  blockSize = 64;
+
+  constructor(key: number[] | string) {
+    key = Hash.toArray(key, 'hex');
+    // Shorten key, if needed
+    if (key.length > this.blockSize) {
+      key = new Hash.SHA1().update(key).digest();
+    }
+
+    // Keys shorter than block size are padded with zeros on the right
+    let i;
+    for (i = key.length; i < this.blockSize; i++) {
+      key.push(0);
+    }
+
+    for (i = 0; i < key.length; i++) {
+      key[i] ^= 0x36;

```

Comment:
Consider 0x36 as a constant

Reply:
What I did for `SHA1HMAC` is: I've made an exact copy of the code from `ts-sdk` [SHA256HMAC](https://github.com/bitcoin-sv/ts-sdk/blob/8212c9d1cfce772d26de5ae8eff195ebc85c4cc4/src/primitives/Hash.ts#L1108-L1190) and adjust it to different `SHA` algorithm. 

I suppose that ts-sdk team will embed this class into the SDK so this file will be deleted. 
So, for now I'd prefer not to "improve" this code if it works.